project2/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Anton|Cabin|Fjalla+One|Quicksand&display=swap" rel="stylesheet">
+    <title>destiny2gamecard</title>
```

Comment:
The `title` can include spaces and should, to make it more clear for your users:
```suggestion
    <title>Destiny 2 game card</title>
```